hydra/_internal/instantiate/_instantiate2.py

```diff
@@ -45,7 +47,7 @@ def _extract_pos_args(*input_args: Any, **kwargs: Any) -> Tuple[Any, Any]:
     return output_args, kwargs
 
 
-def _call_target(_target_: Callable, *args, **kwargs) -> Any:  # type: ignore
+def _call_target(_target_: Callable, partial: Optional[bool], *args, **kwargs) -> Any:  # type: ignore
```

Comment:
Could we get away with `bool` instead of `Optional[bool]`?
(Same for the signature of the `instantiate_node` function below)

Reply:
the  reason it is `None` is a bit nuanced. with configs like `_recursive_`,`_convert_`,   a child node  will always inherit the  parent node config if the config is not specifically set in the child node. But  `_partial_` is different, i think it only makes sense to  set `_partial_` per node, which means if `_partial_` is not set in  a node,  it should default to False instead of inherit that from the parent.

Reply:
I agree that `_partial_` should be set per-node...
 why not make the default `partial=False` instead of `partial=None`?

Reply:
you are right, Pr updated! :)